app/javascript/dashboard/i18n/locale/en/inboxMgmt.json

```diff
@@ -83,7 +83,7 @@
         },
         "CHANNEL_GREETING_TOGGLE": {
           "LABEL": "Enable channel greeting",
-          "HELP_TEXT": "Automatically send a greeting message after the contact's first message in a conversation.",
+          "HELP_TEXT": "Instantly engage customers with automated greeting messages.",
```

Comment:
```suggestion
          "HELP_TEXT": "Automatically send greeting messages when customers start a conversation.",
```

Reply:
@pranavrajs @iamsivin Suggested a new line. Please see.

Reply:
@Hricha-Shandily  this ended up as the original copy :D.  But it wasn't very clear for users as the greeting message was not sent when someone clicked on the start conversation button.  

The greeting message is only sent once the user starts a conversation and then sends a message into the conversation.  We would have to capture that.

Reply:
CC: @pranavrajs @sojan-official

Reply:
> The greeting message is only sent once the user starts a conversation and then sends a message into the conversation. We would have to capture that.

"Auto-send greeting messages when customers start a conversation and send their first message."